meta-lmp-base/recipes-extended/ostree/ostree/0007-sysroot-deploy-systemd-boot-efi-to-ESP-partition.patch

```diff
@@ -0,0 +1,102 @@
+From 8c3bce8d0f9a9148604148d15472807ef28aafc1 Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Thu, 31 Oct 2024 15:16:29 +0100
+Subject: [PATCH] sysroot: deploy systemd-boot efi to ESP partition
+
+Add support for deploying systemd-boot to ESP partition.
+Considering that default deployment has always index == 0, the filenames
+in ESP for efi binaries are generated in this way:
+deployment index       path
+0:                     /boot/EFI/BOOT/bootx64.efi
+1:                     /boot/EFI/BOOT/bootx64_1.efi
+2:                     /boot/EFI/BOOT/bootx64_2.efi
+
```

Comment:
Could you please add Upstream-Status.

With this, LGTM

Reply:
fixed, thanks